yt_dlp/extractor/bilibili.py

```diff
@@ -1929,9 +1920,9 @@ def _extract_video_metadata(self, url, video_id, season_id):
 
         # XXX: webpage metadata may not accurate, it just used to not crash when video_data not found
         return merge_dicts(
-            self._parse_video_metadata(video_data), self._search_json_ld(webpage, video_id, fatal=False), {
-                'title': self._html_search_meta('og:title', webpage),
-                'description': self._html_search_meta('og:description', webpage)
```

Comment:
keep these as fallbacks unless you are sure they aren't relevant anymore. Often sites have slightly different page structures in diff countries.

Reply:
As suggested, I've added these methods again as fallback.